js/background.js

```diff
@@ -277,4 +280,38 @@
             timestamp: timestamp, source: pushMessage.source
         });
     }
+
+    var App = Backbone.Model.extend({
```

Comment:
The short is that backbone provides some really nice class semantics, which  e.g. the initialization routine. Maybe a case of YAGNI? My thought was to build out a global 'app' class for mocking and testing.